src/test/datascience/notebook/remoteNotebookEditor.vscode.common.test.ts

```diff
@@ -39,7 +39,7 @@ import {
 } from '../../../notebooks/controllers/types';
 
 /* eslint-disable @typescript-eslint/no-explicit-any, no-invalid-this */
-suite('DataScience - VSCode Notebook - Remote Execution', function () {
+suite('Remote Execution @kernelCore', function () {
```

Comment:
curious about tagging a suite vs tagging a test - do all tests in the suite effectively get that tag applied?

Reply:
Not sure what you mean.
Suite is the top level container, and when using mocha `grep` we can filter tests via the grep.
The grep is applied top to bottom,
I.e. if i want to include tests that contain the work `@kernelCore`, then i can include that at the `suite` level or individual `test` level.
If the suite contains the text, then the whole suite is included (which effectively includes the whole test).

Today you can run all of the tests within a given suite

Reply:
ah, I missed that the mocha grep was being used on the tags. that makes sense, thanks